packages/amplication-server/src/core/resource/dto/ResourceCreateInput.ts

```diff
@@ -29,4 +30,9 @@ export class ResourceCreateInput {
 
   @Field(() => ConnectGitRepositoryInput, { nullable: true })
   gitRepository?: ConnectGitRepositoryInput;
+
+  @Field(() => EnumCodeGenerator, {
+    nullable: false,
```

Comment:
Shouldn't this be `nullable : true` ?

Reply:
see my previous reply

Reply:
I guess it's confusing since below the actual member `codeGenerator` is defined with a `?` but if that's the right way to define it, feel free to resolve this comment

Reply:
You are correct, but it means that the property is not actually required on the type, but it is required in the API via gql
we have a fallback for undefined value, so we can support undefined, but we don't want to increase the gap so we make it required from the client